etl/steps/data/garden/who/2025-04-07/gho_smoking.meta.yml

```diff
@@ -107,4 +126,77 @@ tables:
 
+
+          ##### Method of measurement
+          Prices provided in different quantities (or volume sizes for e-liquids) were standardized to 20 sticks for cigarettes and Heated Tobacco Products (HTPs).
+
+          Prices in local currency were divided by the IMF’s World Economic Outlook implied PPP conversion rates for 2022 for the conversion in international dollars at purchasing power parity (PPP).
+
+          Prices in local currency were also divided by the IMF’s exchange rate conversion of national currency per US$ end of period data for July 2022 for the conversion in US$ at official exchange rates.
+
+          ##### Limitations
+          In most countries, national level brand-specific price statistics were not available, thus the retail price data were collected from one supermarket/hypermarket or kiosk/newsagent/tobacconist/independent food store usually in the capital city of the country (where survey respondents were mostly located), therefore potentially not being nationally representative.
+      tobacco_ad_ban:
+        title: Bans on tobacco advertising
+        unit: ""
+        short_unit: ""
+        description_short: |-
+          The level of bans on tobacco advertising are defined as follows: 1) no ban, or ban that does not cover national TV, radio and print media advertising; 2) ban on national TV, radio and print media advertising only; 3) ban on national TV, radio and print media, as well as on some, but not all other forms of advertising and 4) complete ban on all forms of advertising.
+        presentation:
+          grapher_config:
+            note: This indicator includes both direct advertising such as TV, radio, print media and billboard ads and indirect advertising, such as sponsorships, product placement or promotional discounts.
+      cig_afford_pct_gdp:
+        title: Affordability of cigarettes (as a share of GDP per capita)
+        unit: "%"
+        short_unit: "%"
+        description_short: |-
+          The price of 100 packs of cigarettes as a share of GDP per capita. This price is based on the most-sold brand of cigarettes in a given country and includes taxes. Lower values indicate that cigarettes are more affordable.
+      tobacco_help_quit:
+        title: Support to help quit smoking
+        unit: ""
+        short_unit: ""
+        description_short: |-
+          National support for quitting smoking or tobacco use has different levels:
+          - Countries in dark blue offer a national quitline and cover costs for nicotine replacement therapy or other services.
```

Comment:
I'd change the last bit to 'nicotine replacement therapy and other cessation services'.